.github/workflows/ci.yml

```
@@ -82,6 +82,33 @@ jobs:
       - name: Run tests
         run: composer test
 
+  static-analysis:
+    name: Static Analysis
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '7.3']
```

I know this kind of review sucks but there's one more space here 🙃 

```suggestion
        php: ['7.3']
```